src/Microsoft.DocAsCode.Dfm/MarkdownValidators/MarkdownValidatorBuilder.cs

```diff
@@ -82,6 +125,30 @@ public IMarkdownTokenRewriter Create()
                 });
         }
 
+        private IEnumerable<MarkdownTagValidationRule> GetEnabledTagRules()
+        {
+            foreach (var item in _tagValidators)
+            {
+                if (item.FullId != null)
+                {
+                    MarkdownValidationRule rule;
+                    if (_validators.TryGetValue(item.FullId, out rule) ||
+                        _validators.TryGetValue(item.Category, out rule))
```

Comment:
In new design, rule always map to contract names of markdown token validators.

---
In reply to: [72757355](https://github.com/dotnet/docfx/pull/545#discussion_r72757355) [](ancestors = 72757355,72756801)